### ISSUE TITLE
Request for Pretrained Models & Test Configs on Datasets

### ISSUE THREAD

**Issue by ShaohuaL** [deborahLevy130/seathru_NeRF#15]
Could you kindly share the pretrained models for these datasets along with the test set configurations used in your published paper?

**Comment by deborahLevy130** [comment]
Hello,
I will try to upload them as soon as possible. I'm not working at the university so I have limited access.  Hope to upload it soon.

**Comment by ShaohuaL** [comment]
Thank you! In particular, I would like to know the names of the images used for training and the names of the images used for testing for each scene in the dataset. It would be better if I could get the camera track of the rendered videos published in the paper, thank you very much.

**Comment by deborahLevy130** [comment]
Hello, 
Sorry for the late reply. You can find the camera track creation in <camera_utils.generate_spiral_path>.
As for the names for each scene of the test images:
Japanese gardens: MTN_1090, MTN_1098 , MTN_1106
Curacao: MTN_1288, MTN_1296, MTN_1034
IUI3: MTN_5928, MTN_5911, MTN_5903, MTN_5894
Panama: MTN_1547, MTN_1539, MTN_1529

**Comment by ShaohuaL** [comment]
> Hello, Sorry for the late reply. You can find the camera track creation in <camera_utils.generate_spiral_path>. As for the names for each scene of the test images: Japanese gardens: MTN_1090, MTN_1098 , MTN_1106 Curacao: MTN_1288, MTN_1296, MTN_1034 IUI3: MTN_5928, MTN_5911, MTN_5903, MTN_5894 Panama: MTN_1547, MTN_1539, MTN_1529

Thank you for your sincere reply. Could you please provide the pretrained model corresponding to these **four scenes**? This is very important to me. Thank you! @deborahLevy130
